packages/wrapped-keys/src/lib/service-client/utils.ts

```diff
@@ -104,29 +105,40 @@ async function getResponseJson<T>(response: Response): Promise<T | string> {
   }
 }
 
+export function generateRequestId(): string {
+  return Math.random().toString(16).slice(2);
+}
```

Comment:
Is the idea to keep the utility function within the `wrapped-keys` package instead of reusing the one in `lit-core`, cur should `wrapped-keys` and `lit-core` import it from the `misc` package?

Reply:
I noticed that the method was made `protected` [recently](https://github.com/LIT-Protocol/js-sdk/pull/640/files) and it was simple enough that the duplication didn't feel like an issue worthy enough of making it part of our public API again...